src/Lecture2.hs

```diff
@@ -97,7 +115,9 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+dropSpaces :: [Char] -> [Char]
+dropSpaces = filter (not . isSpace)
```

Comment:
That's an almost correct solution! 👏🏻 
However, it hangs on infinite lists. So if I do something like this:

```haskell
ghci> take 4 (dropSpaces ("foo" ++ repeat ' '))
```

I'll see that this function will hang.